src/stactools/cop_dem/stac.py

```diff
@@ -108,6 +110,34 @@ def create_item(
     item.add_asset("data", data_asset)
     RasterExtension.ext(data_asset, add_if_missing=True).bands = [data_bands]
 
+    # Make a list of /AUXFILES assets
```

Comment:
Classification URI should be added to stac_extensions here

Reply:
That's what I was asking about earlier when I said Pystac doesn't have the Classification ext, and I couldn't find any examples https://github.com/stactools-packages/landsat/blob/9f595a9d5ed6b62a2e96338e79f5bb502a7d90d0/examples/landsat-c2-l2/collection.json#L59-L64 uses it but doesn't list it anywhere. Is the only way to implement by adding it to Pystac as in this [tutorial](https://pystac.readthedocs.io/en/stable/tutorials/adding-new-and-custom-extensions.html)?

Reply:
I manually added the URI, does that work?

Reply:
You do have to manually add the URI, but also to the Item (the code you added was only added to the Collection). 

I think this will work around line 154:

```
  item.stac_extensions.append("https://stac-extensions.github.io/classification/v1.1.0/schema.json")
```

Looks like this is what landsat is doing here     item.stac_extensions.append(CLASSIFICATION_EXTENSION_SCHEMA)

Reply:
Also, actually implementing the ClassificationExtension would be a nice thing to have, and it doesn't have to be implemented in the pystac package. When I did the GridExtension, I just put it in stactools-sentinel2 and then ported it to stactools, and then when a newer version of stactools was released, replaced it it from that package with the pystac one.

Reply:
I see how Landsat did it now (though they missed listing it on the collection). Done.

I'll open a ticket as a TODO for later to create the ClassificationExtension on pystac. I don't want me having to learn/understand how to do that correctly to hold up this feature/PR.

Reply:
Sounds good. I mostly wanted to make it clear that the extension doesn't **have** to be part of pystac.